### PR TITLE
fan-page

### PR DESCRIPTION
**Pull request by addisonz**
Keeping all styles in an external CSS file makes it more organized so you know where your CSS is making it easier to read.You are able to edit the file and find the spot where you want to edit.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
Hi Addison, 

Great job on your fan-page! The only I didn't happen to see was main content at 80% width. Overall nice job!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        </ul>
+        <h1>IGN</h1>
+            <h2>Game Reviews.</h2>
+        <h3>Game Wikis.</h3>
+    </main>
+
+    <footer>
+        <h4>Contact Information</h4>
+        <ul>
+            <li>E-mail: Addison@Zirbes.com</li>
```
Comment (mbMosman) [inline index.html]: You should still make this an email link using the href and the mailto.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        </ul>
+        <p>Background image courtesy of <a href="http://7-themes.com/data_images/out/67/6999936-cool-websites-background.jpg">7-themes</a>
+            <br>Image icon courtesy of <a href="http://tips.betdownload.com/cf/images/ndh/2013/12/create-a-shutdown-or-restart-your-desktop-icons-1.jpg">betdownload</a></p>
```
Comment (mbMosman) [inline index.html]: Nice work with the image credits!

File: style.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    box-shadow: 0px 5px 5px;
+}
+
+h4 {
+    margin-top: 100px;
+}
+
+div {
+    background-color: lightYellow;
+    background-image: url(backgroundelementimage.jpg);
```
Comment (mbMosman) [inline style.css]: you should put your images into an image folder, so this would be
`background-image: url("images/backgroundelementimage.jpg");`

**Pull request closed without merging**